.github/workflows/performance-testing.yml

```
@@ -31,13 +31,13 @@ jobs:
         run: npm install -g pnpm
 
       - name: Install dependencies
-        run: make install
+        run: pnpm install --frozen-lockfile
         if: |
           steps.cache-pnpm-dependencies.outputs.cache-hit != 'true' ||
```

actionlint (1.7.4)

property "cache-pnpm-packages" is not defined in object type {cache-pnpm-dependencies: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)